example/spec/serial/ingestPdrWithNodeNameFtpSpec.js

```diff
@@ -351,17 +352,20 @@ describe('Ingesting from PDR', () => {
           // delete ingested granule(s)
           await Promise.all(
             finalOutput.payload.granules.map(async (g) => {
+              const id = constructCollectionId(finalOutput.meta.collection.name, finalOutput.meta.collection.version);
               await waitForApiStatus(
                 getGranule,
                 {
                   prefix: config.stackName,
                   granuleId: g.granuleId,
+                  collectionId: id,
                 },
                 'completed'
               );
               await deleteGranule({
                 prefix: config.stackName,
                 granuleId: g.granuleId,
+                collectionId: id,
```

Comment:
If we do the above this would just be 
```
await deleteGranule({
                prefix: config.stackName,
                granuleId: g.granuleId,
                collectionId
})
```